cluster/cluster.go

```diff
@@ -33,6 +33,26 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+// ClusterPeer helps consumers gather information about the peer(s) in the cluster
+type ClusterPeer interface {
```

Comment:
I had to place this in the cluster package as both API `v1` and `v2` consume this interface. Once we remove the v1 code I'll move to where it's used in the API package.

Reply:
Also, I'm happy to take suggestions on a better name.